rna_design/rna_synthesizer_n.py

```diff
@@ -0,0 +1,316 @@
+#!/usr/bin/env python
```

Comment:
_n is the one with initializer, finally replacing the older version after this review.